main.js

```diff
@@ -17,16 +17,39 @@ var maxRange = document.querySelector('#max-range');
 var minSpan = document.querySelector('#min-span');
 var maxSpan = document.querySelector('#max-span');
 var updateButton = document.querySelector('#update-button');
+var errorIcon = document.querySelector('#error');
+var winnerName = document.querySelector('#winner-name');
+var closeWinnerOutputButton = document.querySelector('#close-winner-output');
+var winnerOutput = document.querySelector('.winner-output');
+var winnerOutputSection = document.querySelector('.container2');
 
 submitButton.disabled = true;
 clearButton.disabled = true;
 resetButton.disabled = true;
+updateButton.disabled = true;
 
 document.addEventListener('keyup', enableSubmit);
 document.addEventListener('keyup', enableClear);
+document.addEventListener('keyup', checkRange);
 clearButton.addEventListener('click', clearInputs);
 submitButton.addEventListener('click', updateLatestGuess);
 updateButton.addEventListener('click', updateRange);
```

Comment:
This doesn't do anything now, if you remove the updateRange function

Reply:
There are some functions or variables throughout that I made, but eventually not using.  So, that is why there might be some parts that are unresponsive.  I was going to clean it up when we do our review and clean up.

Reply:
Ignore this anyway, I just realized you moved the updateRange function further up the code. My mistake!

Reply:
No problem!